lua/net.lua

```diff
@@ -8,6 +8,12 @@ return function (gatekeeper_server)
 	-- These parameters should likely be initially changed.
 	local log_level = staticlib.c.RTE_LOG_DEBUG
 
+	-- The user that will run Gatekeeper after it boots.
+	local user = "gatekeeper"
```

Comment:
1. Drop the comment. Instead, update the [documentation](https://github.com/AltraMayor/gatekeeper/wiki/Network) to include this parameter.

2. Have no blank line between variables `log_level` and variable `user`.

3. Update the line of variable `user` to `local user --= "gatekeeper"` to make it initialize to `nil`. Having a user is useful for production, but we don't need it for development.